rust/routee-compass-core/src/util/unit/energy.rs

```diff
@@ -41,6 +41,14 @@ impl From<(EnergyRate, Distance)> for Energy {
     }
 }
 
+impl Div for Energy {
```

Comment:
what was wrong with derive_more::Div?

Reply:
Trying to divide an energy value by another energy value led to this compiler error:

```console
cannot divide `ordered_float::OrderedFloat<f64>` by `routee_compass_core::util::unit::Energy`
the trait `std::ops::Div<routee_compass_core::util::unit::Energy>` is not implemented for `ordered_float::OrderedFloat<f64>`
the following other types implement trait `std::ops::Div<Rhs>`:
  <ordered_float::OrderedFloat<T> as std::ops::Div>
  <ordered_float::OrderedFloat<T> as std::ops::Div<T>>
  <ordered_float::OrderedFloat<T> as std::ops::Div<&'a T>>
  <ordered_float::OrderedFloat<T> as std::ops::Div<&'a ordered_float::OrderedFloat<T>>>
  <&'a ordered_float::OrderedFloat<T> as std::ops::Div<ordered_float::OrderedFloat<T>>>
  <&'a ordered_float::OrderedFloat<T> as std::ops::Div<T>>
  <&'a ordered_float::OrderedFloat<T> as std::ops::Div<&'a T>>
required for `routee_compass_core::util::unit::Energy` to implement `std::ops::Div`
```